### ISSUE TITLE
sys/timeb.h is not available on OpenBSD

### ISSUE THREAD

**Issue by juanfra684** [facebook/zstd#319]
`timeb.h` was removed from OpenBSD a few years ago. `zbufftest.c` doesn't build without it:

```
zbufftest.c: In function 'FUZ_GetMilliStart':
zbufftest.c:76:18: error: storage size of 'tb' isn't known
     struct timeb tb;
                  ^
zbufftest.c:78:5: warning: implicit declaration of function 'ftime' [-Wimplicit-function-declaration]
     ftime( &tb );
     ^
zbufftest.c:76:18: warning: unused variable 'tb' [-Wunused-variable]
     struct timeb tb;
                  ^
gmake: *** [Makefile:112: zbufftest] Error 1
```

**Comment by Cyan4973** [comment]
Good point.
This can be changed to only rely on <time.h>, so it will be changed

**Comment by Cyan4973** [comment]
`zbufftest` and `zstreamtest` no longer depend on `timeb.h`
in latest update on `dev` branch

**Comment by ad-m** [comment]
Is it ready to close?

**Comment by Cyan4973** [comment]
Yes, 
though I typically close an issue when the associated fix reaches release status

**Comment by juanfra684** [comment]
There is a remaining `#include <sys/timeb.h>` in `tests/fuzzer.c`.

**Comment by Cyan4973** [comment]
yes, right,
this is just an oversight, `<timeb.h>` is not necessary in fact.
removed

**Comment by Cyan4973** [comment]
fixed in v1.1.0
